.github/workflows/build-and-test-macos.yaml

```diff
@@ -40,6 +40,12 @@ jobs:
       matrix:
         os: ["macos-13", "macos-14", "macos-15"]
         otp: ["24", "25", "26", "27", "28"]
+        cmake_opts_other: [""]
+
+        include:
+        - os: "macos-13"
```

Comment:
macos-13 is being deprecated on GH next month.. so maybe use macos-14 or 15?

Reply:
Sure. This really belongs to #1770. I picked macos-13 to have an Intel runner but there probably is a more recent alternative. Not sure #1770 JIT works on 26